samples/model/my-model-job.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: Model
+metadata:
+  name: my-xgboost-model
+spec:
+  modelName: my-xgboost-model
+  primaryContainer:
+    containerHostname: xgboost
+    modelDataURL: s3://<YOUR BUCKET>/sagemaker/model/xgb-churn-prediction-model.tar.gz
+    #source-data-bucket-718865417152-us-west-2
+    image: <YOUR ECR IMAGE> # The URL and tag of your ECR container  
+    # 433757028032.dkr.ecr.us-west-2.amazonaws.com/xgboost:1
+    environment:
+      my_var: my_value
+      my_var2: my_value2
```

Comment:
nit: Remove these environment variables. Or at least rename them to resemble something more useful